my-app/src/SubmitSearch.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * Child Class 2 - renders search/refresh buttons
+ * Stateless Component Class
+ */
+
+import React from 'react';
+
+export const SubmitSearch = (props) => {
+    
+    console.log("3. I render search and refresh buttons");
+    let searchStatus = props.searchStatus;
+
+    if (!searchStatus) {
```

Comment:
While this is correct, it could be simplified to `if (!props.searchStatus)`, and the `else if` converted to `else`. A similar tweak could be made for the `SortResults` file